infrastructure/ArmTemplates/antareskvtemplate.json

```diff
@@ -370,9 +370,19 @@
                   "subStatus": 0,
                   "count": 100,
                   "timeInterval": "00:10:00"
-                }
+                },
+                {
+                  "status": 502,
+                  "subStatus": 0,
```

Comment:
>subStatus [](start = 19, length = 9)

We should just remove the substatus completely so we get data whenever a 502 happens so we are covered for all